### ISSUE TITLE
Not connected to Golem Network.  Check seed parameters.

### ISSUE THREAD

**Issue by StuartFeldt** [golemfactory/homebrew-golem#7]
OSX 10.12.2.

I have tried port forwarding of both 40102 and 8333 to my mac, as well as used known seeds from [here](https://github.com/golemfactory/golem/wiki/Installation#known-seeds).

on startup:

```
Failed to import scrypt. This is not a fatal error but does
mean that you cannot create or decrypt privkey jsons that use
scrypt

I0401 11:23:10.654700 cmd/utils/flags.go:613] WARNING: No etherbase set and no accounts found as default
I0401 11:23:10.654848 ethdb/database.go:83] Allotted 128MB cache and 1024 file handles to /Users/stu.feldt/Library/Ethereum/testnet/geth/lightchaindata
I0401 11:23:10.664883 ethdb/database.go:176] closed db:/Users/stu.feldt/Library/Ethereum/testnet/geth/lightchaindata
I0401 11:23:10.667965 node/node.go:176] instance: Geth/v1.5.9-stable-abc71f5f/darwin/go1.8
I0401 11:23:10.667999 ethdb/database.go:83] Allotted 128MB cache and 1024 file handles to /Users/stu.feldt/Library/Ethereum/testnet/geth/lightchaindata
I0401 11:23:10.695963 core/genesis.go:95] Genesis block already in chain. Writing canonical number
I0401 11:23:10.696034 eth/backend.go:276] Successfully wrote custom genesis block: 41941023680923e0fe4d74a34bdac8141f2540e3ae90623718e47d66d1ca4a2d
I0401 11:23:10.698043 light/lightchain.go:172] Last header: #668338 [42104933…] TD=492605733744021
I0401 11:23:10.699268 p2p/server.go:340] Starting Server
I0401 11:23:10.701925 les/backend.go:191] WARNING: light client mode is an experimental feature
I0401 11:23:10.701978 p2p/server.go:608] Listening on [::]:30303
I0401 11:23:10.703468 node/node.go:341] IPC endpoint opened: /Users/stu.feldt/Library/Ethereum/testnet/geth.ipc
I0401 11:23:10.726168 p2p/nat/nat.go:111] mapped network port tcp:30303 -> 30303 (ethereum p2p) using NAT-PMP(192.168.86.1)
ERROR    [golem.docker.config_manager        ] Couldn't read CPU affinity: 'Process' object has no attribute 'cpu_affinity'
I0401 11:23:11.127491 eth/downloader/downloader.go:326] Block synchronisation started
I0401 11:23:12.485892 core/headerchain.go:342] imported    2 headers in 804.632ms. #668340 [249c431c… / 0467f7e6…]
I0401 11:23:12.805500 ssdp.go:63] ssdp: got unexpected search target result "upnp:rootdevice"
I0401 11:23:12.805523 ssdp.go:63] ssdp: got unexpected search target result "uuid:2f402f80-da50-11e1-9b23-001788182a5b"
I0401 11:23:12.805531 ssdp.go:63] ssdp: got unexpected search target result "urn:schemas-upnp-org:device:basic:1"
I0401 11:23:12.805539 ssdp.go:63] ssdp: got unexpected search target result "upnp:rootdevice"
I0401 11:23:12.805547 ssdp.go:63] ssdp: got unexpected search target result "uuid:2f402f80-da50-11e1-9b23-001788182a5b"
I0401 11:23:12.805555 ssdp.go:63] ssdp: got unexpected search target result "urn:schemas-upnp-org:device:basic:1"
HyperG is ready

Failed to import scrypt. This is not a fatal error but does
mean that you cannot create or decrypt privkey jsons that use
scrypt

du: illegal option -- b
usage: du [-H | -L | -P] [-a | -s | -d depth] [-c] [-h | -k | -m | -g] [-x] [-I mask] [file ...]
```

<img width="1192" alt="screen shot 2017-04-01 at 11 27 27 am" src="https://cloud.githubusercontent.com/assets/427126/24580483/3bc8890e-16ce-11e7-9c81-8a00e15f765f.png">

**Comment by trublade** [comment]
Ive got the same issue on my Mac

**Comment by gborowik** [comment]
see here: 
https://blog.golemproject.net/announcing-the-pirx-testnet-and-a-windows-release-e1e0067e95bf
and
https://blog.golemproject.net/pre-brass-golem-0-6-0-released-a6501ed8f178
